### DISCUSSION TITLE
firmware definition

### DISCUSSION THREAD

**Discussion by chris-1243** [darkxst/silabs-firmware-builder#44] [General]
In the Dongle-E repository does someone may tell me what is the difference between such firmware ?

ncp-uart-hw-v7.3.2.0-zbdonglee-none-115200.gbl
ncp-uart-hw-v7.3.2.0-zbdonglee-115200.gbl

What is the meaning of the "none" attribute?

**Comment by darkxst** [comment]
>  ncp-uart-hw-v7.3.2.0-zbdonglee-115200.gbl

This is the default firmware via web flasher and includes software flow control (xon/xoff)

> ncp-uart-hw-v7.3.2.0-zbdonglee-none-115200.gbl
> What is the meaning of the "none" attribute?

This one flow control turned off completely.
